{"name": "Негативный тест принятия заказа. Не передаем ID заказа", "status": "passed", "description": "Передаем пустое значение ID заказа. Ожидаем 404 статус код и ошибку в теле ответа", "steps": [{"name": "Принять заказ", "status": "passed", "parameters": [{"name": "order_id", "value": "''"}, {"name": "courier_id", "value": "<function Helper.random_number_from_tests at 0x0000021D473FCF40>"}], "start": 1715183392007, "stop": 1715183392414}], "start": 1715183392007, "stop": 1715183392414, "uuid": "95df0143-54ef-4154-88da-be641becc260", "historyId": "c52ac04ef20efe9977b7b8f1857b8a2e", "testCaseId": "c52ac04ef20efe9977b7b8f1857b8a2e", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_empty_order_id_failed_accept_order", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-86Q2G53"}, {"name": "thread", "value": "18992-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}